### PR TITLE
Issue166 ts3 official

### LINKED ISSUE #166: TS3 Rock-On Deprecated
Additional information can be seen on the forums:

https://forum.rockstor.com/t/teamspeak-rock-on-dead/5282

### PR DESCRIPTION
**Pull request by FroggyFlox**
Fix #166 

This PR updates the Teamspeak3 Rock-on to use the official docker image: [https://hub.docker.com/_/teamspeak](https://hub.docker.com/_/teamspeak)

This change is currently **required** as the current image has already been deprecated. See Issue #166 for details and discussion on naming convention.

This has been tested to work on Rockstor-3.9.1-16 by myself and confirmed by the issue author @Noggin01.

@phillxnet, this PR is ready for your attention.

### CODE REVIEW

**Comment by phillxnet** [comment]
@FroggyFlox Note that I didn't test the Teamspeak function itself, just our end of things.

File: teamspeak3.json [hunk 1]
```diff
@@ -28,29 +28,24 @@
...
-    	    "more_info": "<h4>You need to get the previliged key using system console to get serveradmin access</h4></p>1. Open system console (don't forget to start the service) or use SSH</p>2. cd /mnt2/[sharename]/serverfiles/logs</p>3. grep \"token\" ./*",
+        "description": "VoIP software designed with security in mind, featuring crystal clear voice quality, endless customization options, and scalabilty up to thousands of simultaneous users.<p>Based on the official Teamspeak docker image: <a href='https://hub.docker.com/_/teamspeak' target='_blank'>https://hub.docker.com/_/teamspeak</a>.",
+    	"more_info": "<h4>You need to get the privileged key using system console to get serveradmin access</h4></p>1. Open system console (don't forget to start the service) or use SSH</p>2. docker logs Teamspeak3_official | grep loginname</p><br><p>If it's the second or more start of this Rock-on, please use the following instead:</p> docker exec Teamspeak3_official sh -c 'cat logs/*.log | grep token'</p>",
```
[the remark this answers is not in the document]
Comment (FroggyFlox) [reply teamspeak3.json]: You're absolutely right! I forgot to revert the name in the `"more_info"` section. I'll correct that.

**Comment by FroggyFlox** [comment]
@phillxnet , the latest commit 3269d41 fixes the mistake you identified.

Thanks again for your review, and for spotting my mistakes.
Note that I, too, haven't tested the Teamspeak functions themselves as noted above, but @Noggin01 did verified that in the issue https://github.com/rockstor/rockon-registry/issues/166#issuecomment-429585447

Please note I'm unsure I dealt correctly with the Github function "Resolve changes" from your review, so I hope I didn't miss a button to fix it somewhere.

**Comment by phillxnet** [comment]
@FroggyFlox Thanks for sorting that out so quickly. And as per my end of the review/resolved thing: it all looks dandy. I have it as now outdated and when I expand it via show resolved it states:
"FroggyFlox marked this conversation as resolved."
So I guess we did that bit as was intended :).

Merging.

**Comment by FroggyFlox** [comment]
Sounds good! Thanks!

**Pull request merged**